bootstrap/consul.pp

```diff
@@ -1,38 +1,38 @@
 file { "/etc/simp":
-	ensure => directory,
+  ensure => directory,
+}
```

Comment:
I'm 90% sure that this is going to conflict with the `simp` module itself.

I get that it's a bootstrap class, but it should be able to run with everything else without issue in general. We might need to change both to an 'ensure_resource' or move them up to a higher level class but dragging along the 'simp' module really shouldn't be a big deal in any environment. It's well namespaced and we're using the materials here anyway.

Reply:
We shouldn't bring in the simp module for libkv; it's intended to also stand completely alone. I think ensure_resource for that directory is the best bet